rna-germline-variant-calling.inputs.json

```diff
@@ -10,9 +22,28 @@
     "gs://broad-references/Homo_sapiens_assembly19_1000genomes_decoy/Homo_sapiens_assembly19_1000genomes_decoy.known_indels.vcf.idx"
   ],
   "RNAseq.annotationsGTF": "gs://gatk-test-data/intervals/star.gencode.v19.transcripts.patched_contigs.gtf",
-  "RNAseq.refFasta": "gs://broad-references/Homo_sapiens_assembly19_1000genomes_decoy/Homo_sapiens_assembly19_1000genomes_decoy.fasta",
-  "RNAseq.refFastaIndex": "gs://broad-references/Homo_sapiens_assembly19_1000genomes_decoy/Homo_sapiens_assembly19_1000genomes_decoy.fasta.fai",
-  "RNAseq.refDict": "gs://broad-references/Homo_sapiens_assembly19_1000genomes_decoy/Homo_sapiens_assembly19_1000genomes_decoy.dict",
-  "RNAseq.wgsCallingIntervalList": "gs://broad-references/Homo_sapiens_assembly19_1000genomes_decoy/Homo_sapiens_assembly19_1000genomes_decoy.whole_genome.interval_list",
-  "RNAseq.inputBam": "gs://gatk-test-data/rna_bam/NA12878_b37/NA12878.bam"
+  
+  "##_COMMENT4": "DOCKERS",
+  "RNAseq.gatk4_docker": "us.gcr.io/broad-gatk/gatk",
+  "RNAseq.star_docker": "ruchim/star:v1",
```

Comment:
I don't quite understand the context for this json, but is this the docker you want to use or is there a more official broad wide star image?

Reply:
Thanks, for catching this. I updated the image in FC to 
quay.io/humancellatlas/secondary-analysis-star:v0.2.2-2.5.3a-40ead6e and will do the same for this json.